File: src/projects/avlg/java/exceptions/InvalidBalanceException.java
package projects.avlg.java.exceptions;

/**
 *<p> InvalidBalanceException is an {@link Exception} that is thrown by the constructor
 * of {@link projects.avlg.java.AVLGTree}
 *
 * <p>You should <b>not</b> edit this class! It is given to you as a resource for your project.</p>

 * @author <a href="https://github.com/jasonfil">Jason Filippou</a>
 *
 * @see EmptyTreeException
*/
public class InvalidBalanceException extends Exception {
    public InvalidBalanceException(String msg){
        super(msg);
    }
}

File: src/projects/avlg/python/exception/InvalidBalanceException.py
"""
<p> <tt>InvalidBalanceException</tt> is an {@link Exception} that is thrown by the constructor 
of {@link projects.avlg.AVLGTree} 

<p>You should <b>not</b> edit this class! It is given to you as a resource for your project.</p> 

@author <a href="https://github.com/jasonfil">Jason Filippou</a> 

@see EmptyTreeException 
"""


class InvalidBalanceException(Exception):
    def __init__(self, message):
        super(InvalidBalanceException, self).__init__(message)